app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split(" ")
+    file_to_copy = command[1]
+    new_file = command[-1]
```

Comment:
You don't follow your logic here. If you know that command has definitely 3 items just pick `command[2]` here and if it doesn't work that means given command wasn't valid and you need to raise some error here.  What if `command` was `"cp path1.txt some_random_text path2.txt some_random_text"`?